fission/src/ui/panels/DebugPanel.tsx

```diff
@@ -60,86 +60,105 @@ const DebugPanel: React.FC<PanelPropsImpl> = ({ panelId }) => {
-                        openPanel("poker")
-                    }}
-                />
-                <Button value={"Test God Mode"} onClick={TestGodMode} />
-                <Button value={"Clear Prefs"} onClick={() => PreferencesSystem.clearPreferences()} />
-                <LabelStyled>Autodesk Platform Services</LabelStyled>
-                <Button
-                    value={"Refresh APS Token"}
-                    onClick={async () =>
-                        APS.isSignedIn() && APS.refreshAuthToken((await APS.getAuth())!.refresh_token, true)
-                    }
-                />
-                <Button
-                    value={"Expire APS Token"}
-                    onClick={() => {
-                        if (APS.isSignedIn()) {
-                            APS.setExpiresAt(Date.now())
-                            APS.getAuthOrLogin()
-                        }
-                    }}
-                />
-                <LabelStyled>Caching Service</LabelStyled>
-                <Button
-                    value={"Print Mira Maps"}
-                    onClick={() => {
-                        console.log(MirabufCachingService.GetCacheMap(MiraType.ROBOT))
-                        console.log(MirabufCachingService.GetCacheMap(MiraType.FIELD))
-                        console.log(hashedMiraRobots)
-                        console.log(hashedMiraFields)
-                    }}
-                />
-                <Button value={"Clear Mira Cache"} onClick={() => MirabufCachingService.RemoveAll()} />
-                <LabelStyled>Code Simulation</LabelStyled>
-                <Button
-                    value={"WS Test"}
-                    onClick={() => {
-                        // worker?.postMessage({ command: 'connect' });
-                        const miraObjs = [...World.SceneRenderer.sceneObjects.entries()].filter(
-                            x => x[1] instanceof MirabufSceneObject
-                        )
-                        console.log(`Number of mirabuf scene objects: ${miraObjs.length}`)
-                        if (miraObjs.length > 0) {
-                            const mechanism = (miraObjs[0][1] as MirabufSceneObject).mechanism
-                            const simLayer = World.SimulationSystem.GetSimulationLayer(mechanism)
-                            simLayer?.SetBrain(new WPILibBrain(mechanism))
+                <Box display="flex" flexDirection={"column"} gap="0.25rem" width={"80%"} margin={"auto"}>
+                    <LabelStyled>Generic</LabelStyled>
+                    <Button
+                        value={"Toasts"}
+                        onClick={() => {
+                            const type: ToastType = ["info", "warning", "error"][Math.floor(Random() * 3)] as ToastType
+                            MainHUD_AddToast(type, type, "This is a test toast to test the toast system")
+                        }}
+                        className="w-full"
+                    />
```

Comment:
Do we need a toast tester to test the toast system still?

Reply:
I think we should keep it because the debug panel is partially meant for testing features that aren't fully integrated, like toasts.